kolibri/core/content/utils/channel_import.py

```diff
@@ -302,7 +303,7 @@ def raw_attached_sqlite_table_import(
         query = """{method} INTO {table} ({destcols}) SELECT {sourcevals} FROM sourcedb.{table} AS source""".format(
             method=method,
             table=dest_table.name,
-            destcols=", ".join(dest_columns),
+            destcols=", ".join(map(lambda c: "'{}'".format(c), dest_columns)),  # Wrap column names in quotes to avoid conflicts with SQL keywords
```

Comment:
This is a change I made because I received an error that complained about the columns being added.

The SQL was similar to:

`... SELECT (col1, col2, order) FROM...`

The error said there was an issue near order.

I thought maybe it was because order is a SQL keyword. When I added the change on this line, which puts all table names in quotes, the error was resolved.